ui-flows/shared/login_demo_account.yml

```diff
@@ -0,0 +1,29 @@
+appId: com.fsck.k9.debug
+---
+- launchApp:
+    clearState: true
+
+# Welcome
+- tapOn:
+    id: "com.fsck.k9.debug:id/next"
+
+## Setup new account
+- tapOn:
+    id: "com.fsck.k9.debug:id/account_email"
+- inputText: "demo@k9mail.example"
+- tapOn:
+    id: "com.fsck.k9.debug:id/next"
+
+## Account name configuration
+- tapOn:
+    id: "com.fsck.k9.debug:id/account_description"
+- inputText: "demo@k9mail"
+- tapOn:
+    id: "com.fsck.k9.debug:id/account_name"
+- inputText: "Demo Account"
```

Comment:
This is the name that is used in outgoing messages. So it should probably be something like "Demo User".